### DISCUSSION TITLE
Projects using Build Onchain Apps Toolkit: LFG 🚀

### DISCUSSION THREAD

**Discussion by Zizzamia** [coinbase/build-onchain-apps#64] [Announcements]
gm ✨ ✨ ✨
If you're using **Build Onchain Apps** library to kick off your project, share the URL and / or GitHub link here.

Don't worry if the project was just a demo for a hackathon or is still partially complete. We are here to learn from the community experience and improve the library together. 🛵

**Comment by jungbok-cb** [comment]
Hi Build Onchain Apps team, 
I am reaching out to express my deep appreciation for this amazing library! Your library has been an invaluable resource for our hackathon project, particularly in learning about smart contracts, deploying them, and integrating them with UI. 
The built-in smart contracts and UI were very helpful, making it easy to test and experiment our web3 application demo. 

I am thrilled to share that our hackathon team won a runner-up prize, significantly aided by your library. Our team created a fiat currency exchange application using this library, and successfully developed a working demo in just 1 day!

Your effort to building such an accessible tool for the community is truly inspiring. I am excited to continuously explore and utilize Build Onchain Apps in the future. Huge thanks again to the entire team for sharing your outstanding work!

**Comment by michaelcohen716** [comment]
Great experience for me using BOAT to build [Crypto Composite](https://www.cryptocomposite.xyz)!

**Comment by thseeding** [comment]
hey guys i love seeing these comments. any one who built anything using this template find any vulnerabilities ? just want to make sure beofre diving in!
